File: app/src/main/java/id/niteroomcreation/archcomponent/CustomApplication.java
package id.niteroomcreation.archcomponent;

import android.app.Application;
import android.content.Context;

/**
 * Created by monta on 10/05/21
 * please make sure to use credit when using people code
 **/
public class CustomApplication extends Application {

    private static Context mContext;

    public static Context getAppContext() {
        return CustomApplication.mContext;
    }

    @Override
    public void onCreate() {
        super.onCreate();

        CustomApplication.mContext = getApplicationContext();
    }
}

File: app/src/main/java/id/niteroomcreation/archcomponent/CustomApplication.kt
package id.niteroomcreation.archcomponent

import android.app.Application
import android.content.Context

/**
 * Created by monta on 10/05/21
 * please make sure to use credit when using people code
 */
class CustomApplication : Application() {

    override fun onCreate() {
        super.onCreate()
        context = applicationContext
    }

    companion object {

        private lateinit var context: Context

        fun getContext() = context
    }
}